### PR TITLE
tutorials

### PR DESCRIPTION
**Pull request by ThatGalNatalie**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Aims to improve and convert the current tutorials into md

### CODE REVIEW

File: src/pages/Tutorials/Markdown/Flow-Run-Tutorial.md
Comment (cicdw) [inline src/pages/Tutorials/Markdown/Flow-Run-Tutorial.md:14]: We should host these images somewhere more permanent / that we have more control over rather than github

File: src/pages/Tutorials/Markdown/Universal-Deploy.md
Comment (cicdw) [inline src/pages/Tutorials/Markdown/Universal-Deploy.md:9]: I noticed the renderer interpreted both `pip` and `conda` as full code blocks instead of inline code - any idea why that is happening?

**Review by cicdw — COMMENTED** [review]
Sorry, I know this is still a draft but I'm really excited for these changes so wanted to poke around a little!

File: src/pages/Tutorials/Markdown/Flow-Run-Tutorial.md
Comment (ThatGalNatalie) [reply src/pages/Tutorials/Markdown/Flow-Run-Tutorial.md:14]: Yeah this was just to test out the md file for now but it will definitely be hosted somewhere else

File: src/pages/Tutorials/Markdown/Universal-Deploy.md
Comment (ThatGalNatalie) [reply src/pages/Tutorials/Markdown/Universal-Deploy.md:9]: After digging into this more, it appears to be a CSS issue. Currently, I'm applying a set of styles to all the `code` elements so it makes it more of a code block instead of inline code.

**Review by znicholasbrown — CHANGES_REQUESTED** [review]
This is looking **GREAT** @ThatGalNatalie.

I won't dive into the code just yet but I have some design notes that we should address first (in no particular order):


**Tutorial page**
- [x] need to add padding on the right side of the main content
- [x] page doesn't scroll to a linked anchor element (e.g. when you copy/paste a link, the anchor is removed from the URL and the page doesn't scroll to the location as expected, nor is the section highlighted in the TOC)
- [x] when navigating directly to the `tutorial` route, the initial page is blank - we should either add some helpful content here or at least add a message about selecting a tutorial from the sidebar


**Markdown styling**
- [x] the font size is a little too large compared to other portions of the app. I'm fine with it being a little larger but we should till dial that down a bit
- [x] since we already have examples of code blocks on the `home` page, we should match code block styling to those
- [x] should increase the margin between sections, right now they feel squished
- [x] rendered lists are very dark compared to the rest of the text


**TOC**
- [x] only the text is clickable instead of the entire line
- [x] sidebar feels unnatural on the right side, could we move it to the left?
- [x] when a given tutorial is selected/in view, its contents should expand in the TOC (right now you can have one tutorial selected but a different one expanded)
- [x] the all-caps for the different navigation items makes them a little difficult to read, we should use sentence casing instead


**Non-blockers**
- [x] we should add code highlighting for python blocks in particular - we can import the codemirror python package for that

File: src/pages/Tutorials/Tutorials.vue
Comment (znicholasbrown) [reply src/pages/Tutorials/Tutorials.vue:39]: I really like the mini-variant idea

File: src/pages/Tutorials/Tutorials.vue
Comment (ThatGalNatalie) [inline src/pages/Tutorials/Tutorials.vue:137]: Okay I moved the sidebar back to the left and it's now using the `mini-variant` prop but I feel like the padding is still weird when the screen size gets smaller. I feel like there is a better way of doing this but I'm not sure

File: src/pages/Tutorials/Markdown/Universal-Deploy.md
Comment (ThatGalNatalie) [reply src/pages/Tutorials/Markdown/Universal-Deploy.md]: Okay, I changed the bolds to headings but in the toc, it won't render as a child of a section. Right now, I can't figure out a way to group all the smaller headings and make that a child of all the h1
Screen Shot 2020-11-19 at 1 38 28 PM

Comment (zhen0) [reply src/pages/Tutorials/Markdown/Universal-Deploy.md]: Hmm... Good point.  Maybe we try again with the bold but make sure the spacing is correct.  So it looks more like this:
Screen Shot 2020-11-19 at 1 14 19 PM
Rather than this (which is what I was seeing before):
Screen Shot 2020-11-20 at 12 51 27 PM

File: src/pages/Tutorials/Tutorials.vue
Comment (zhen0) [reply src/pages/Tutorials/Tutorials.vue:141]: Can you share a picture of what you mean here?  I see it overlap a lot on mobile sized screens but not sure that's what you're referring to?

File: src/pages/Tutorials/Markdown/Universal-Deploy.md
Comment (ThatGalNatalie) [reply src/pages/Tutorials/Markdown/Universal-Deploy.md]: I asked Nicholas about this and he said it was okay not to have the smaller headings nested in the toc but I did change all the bolds to h4 
heading

File: src/pages/Tutorials/Tutorials.vue
Comment (ThatGalNatalie) [reply src/pages/Tutorials/Tutorials.vue:141]: So I think what I meant was when it's on mobile screen sizes and the toc is open, the toc will overlap but not if the toc is closed. I think I was just having a hard time making it look good on mobile screens

closed
open

Comment (zhen0) [reply src/pages/Tutorials/Tutorials.vue]: Just took a look and I think you've got it fixed.  It closed on hover/click for me which meant I could read and navigate fine. Also, I can't see many users wanting to use mobile for the tutorials.

**Comment by zhen0** [comment]
This looks good to me.  I'll let Nicholas have final review/approval.

**Review by znicholasbrown — APPROVED** [review]
This LGTM @ThatGalNatalie !!

**Pull request merged**
